reports/README.md

```diff
@@ -0,0 +1,10 @@
+## Authoring krabby reports
```

Comment:
for now, they are. I figured once we start breaking them out and this `dir` is wiped, we would moved it to the readme.

thoughts on that?